src/test/java/com/github/k0kubun/builder/query/graphql/GraphQLTest.java

```diff
@@ -1,20 +0,0 @@
-package com.github.k0kubun.builder.query.graphql;
-
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
-public class GraphQLTest
```

Comment:
Removed this test because it was not covering anything else than what GraphQLQueryBuilderTest did